### PR TITLE
Personal finance dashboard page

### PR DESCRIPTION
**Pull request by ricardoprestes**
## Submission description

This is a simple sample dashboard page for a personal finances app. This page contains balance single view, expenses categories chart, next movimentations and credit card bills.

### CODE REVIEW

File: src/Snppts/Snppts.csproj [hunk 1]
```diff
@@ -6,7 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.App">
```
Comment (jfversluis) [inline src/Snppts/Snppts.csproj]: Please don't change the csproj file :)

**Review by jfversluis — CHANGES_REQUESTED** [review]
See comment

**Comment by ricardoprestes** [comment]
Changes in .csproj reverted

**Review by jfversluis — APPROVED** [review]
Looking awesome! Thank you for your contribution!

**Comment by sthewissen** [comment]
Thanks for contributing to Snppts! Please fill in the following survey to receive your swag :)

https://www.surveymonkey.com/r/88ZQ6G2

**Pull request merged**